ajax/libs/sanctuary/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "sanctuary",
+  "description": "Refuge from unsafe JavaScript",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sanctuary-js/sanctuary.git"
+  },
+  "dependencies": {
+    "sanctuary-def": "0.20.1",
+    "sanctuary-either": "1.2.0",
+    "sanctuary-maybe": "1.2.0",
+    "sanctuary-pair": "1.2.0",
+    "sanctuary-show": "1.0.0",
+    "sanctuary-type-classes": "11.0.0",
+    "sanctuary-type-identifiers": "2.0.1"
+  },
```

Comment:
Dependencies are subject to change from one version to the next. Can `dependencies` safely be omitted?

Reply:
Yup.

Reply:
Thanks. I have removed `dependencies`. :)